ejb3/src/main/resources/schema/jboss-ejb-client_1_3.xsd

```diff
@@ -0,0 +1,330 @@
+  ~ published by the Free Software Foundation; either version 2.1 of
+  ~ the License, or (at your option) any later version.
+  ~
+  ~ This software is distributed in the hope that it will be useful,
+  ~ but WITHOUT ANY WARRANTY; without even the implied warranty of
+  ~ MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+  ~ Lesser General Public License for more details.
+  ~
+  ~ You should have received a copy of the GNU Lesser General Public
+  ~ License along with this software; if not, write to the Free
+  ~ Software Foundation, Inc., 51 Franklin St, Fifth Floor, Boston, MA
+  ~ 02110-1301 USA, or see the FSF site: http://www.fsf.org.
+  -->
+
+<xsd:schema xmlns:xsd="http://www.w3.org/2001/XMLSchema"
+            targetNamespace="urn:jboss:ejb-client:1.3"
+            xmlns="urn:jboss:ejb-client:1.3"
+            elementFormDefault="qualified"
+            attributeFormDefault="unqualified"
+            version="1.3">
+
+    <!-- Root element -->
+    <xsd:element name="jboss-ejb-client" type="jboss-ejb-clientType">
+        <xsd:annotation>
+            <xsd:documentation>
+                Root element for a jboss-ejb-client.xml file
+            </xsd:documentation>
+        </xsd:annotation>
+    </xsd:element>
+
+    <xsd:complexType name="jboss-ejb-clientType">
+        <xsd:annotation>
+            <xsd:documentation>
+                The EJB client configurations
+            </xsd:documentation>
+        </xsd:annotation>
+        <xsd:all>
+            <xsd:element name="client-context" type="client-contextType">
+                <xsd:annotation>
+                    <xsd:documentation>
+                        Configurations that will be used to setup an EJB client context for the
+                        deployment.
+                    </xsd:documentation>
+                </xsd:annotation>
+            </xsd:element>
+        </xsd:all>
+    </xsd:complexType>
+
+    <xsd:complexType name="client-contextType">
+        <xsd:all>
+            <xsd:element name="ejb-receivers" type="ejb-receiversType">
+                <xsd:annotation>
+                    <xsd:documentation>
+                        Configures EJB receivers for the client context
+                    </xsd:documentation>
+                </xsd:annotation>
+            </xsd:element>
+            <xsd:element name="clusters" type="clustersType" minOccurs="0">
+                <xsd:annotation>
+                    <xsd:documentation>
+                        Cluster configurations applicable for this client context
+                    </xsd:documentation>
+                </xsd:annotation>
+            </xsd:element>
+            <xsd:element name="profile" type="profileType" minOccurs="0">
+                <xsd:annotation>
+                    <xsd:documentation>
+                        Cluster configurations applicable for this client context
+                    </xsd:documentation>
```

Comment:
need to change documentation to reflect the purpose of the new element.  Probably something like `Remoten connection profile selected for this client context.`